src/refactor_nrepl/find/symbols_in_file.clj

```diff
@@ -56,8 +56,12 @@
   ([path parsed-ns dialect]
    (util/with-additional-ex-data [:file path]
      (let [dialect (or dialect (core/file->dialect path))
-           file-ns (or (find-ns (symbol (:ns parsed-ns))) *ns*)
-           ns-aliases (if (= dialect :cljs)
+           cljs? (= dialect :cljs)
+           file-ns (or (when-let [s (-> parsed-ns :ns symbol)]
+                         (when-not cljs?
+                           (core/safe-find-ns s true)))
```

Comment:
You don't have to change this, but in general it's nice to send in `:ignore-errors` instead of `true` as it makes the call-sites more readable when you don't have to remember how the truthy flags alter the behavior